city_scrapers/spiders/chi_ssa_38.py

```diff
@@ -0,0 +1,116 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa38Spider(CityScrapersSpider):
+    name = "chi_ssa_38"
+    agency = "Chicago Special Service Area #38 Northcenter"
+    timezone = "America/Chicago"
+    start_urls = ["http://www.northcenterchamber.com/pages/MeetingsTransparency1"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for item in response.css("div#content-167642 li::text"):
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self._parse_location(item),
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        title = "Chamber of Commerce"
+        return title
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        description = ""
+        return description
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        date_str = item.extract()
```

Comment:
It looks like we might be able to simplify this a bit, and we'll also need to handle situations where minutes may be supplied for the time. Haven't tested this, but something like this snippet could work:


```python
item_str = item.extract()
month_day_str = re.search(r"[A-Z][a-z]{2,9} \d{1,2}", item_str).group()

year_str = re.search(r"\d{4}", item_str).group()
if not year_str[:2] == "20":
  year_str = str(datetime.today().year)  # Default to current year

time_match = re.search(r"\d{1,2}(\:\d\d) [apm\.]{2,4}", item_str)  # We want to check for a minutes portion here
time_str = "12 am"
if time_match:
  time_str = time_match.group().replace(".", "")

time_fmt = "%I %p"
if ":" in time_str:
  time_fmt = "%I:%M %p"

return datetime.strptime(f"{month_day_str} {year_str} {time_str}", f"%B %d %Y {time_str}")
```